io/driver/module_unload_load.sh

```diff
@@ -13,13 +13,15 @@
 #
 # Copyright: 2016 IBM
 # Author: Harsha Thyagaraja <harshkid@linux.vnet.ibm.com>
+# Copyright: 2017 IBM
+# Authors: Narasimhan V <sim@linux.vnet.ibm.com>
```

Comment:
example when the module is completely reworked or extended in a such extend that the original work is not that significant anymore.